guides/common/modules/con_provisioning-virtual-machines-on-kvm.adoc

```diff
@@ -20,7 +20,7 @@ Ensure no other DHCP services run on this network to avoid conflicts with {Smart
 For more information about network service configuration for {SmartProxyServer}s, see {ProvisioningDocURL}Configuring_Networking_provisioning[Configuring Networking] in _{ProvisioningDocTitle}_.
 ifdef::satellite[]
 * A {RHEL} server running KVM virtualization tools (libvirt daemon).
-For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/virtualization_getting_started_guide/[{RHEL} 7 Virtualization Getting Started Guide].
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_virtualization/index[_{EL} 8 Configuring and managing virtualization_].
```

Comment:
This links to Red Hat documentation for RHEL so I think this can safely say RHEL.
```suggestion
For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_virtualization/index[_{RHEL} 8 Configuring and managing virtualization_].
```

Reply:
Addressed :+1: